src/edfi_amt_data_lake/api/changeVersion.py

```diff
@@ -29,6 +35,15 @@ def get_change_version_values_from_file(file) -> ChangeVersionValues:
     return ChangeVersionValues('0', '0')
 
 
+def _delete_downloaded_files() -> None:
+    import shutil
+    path = f"{config('CHANGE_VERSION_FILEPATH')}"
+    if not os.path.exists(path):
```

Comment:
We don't need an exception if the path doesn't exist. We are just cleaning the destination folder, if the folder should finish the function because we don't have nothing to delete.

Reply:
If an error occurred when creating the directory, this exception helps us to know if the directory creation failed.